src/values/TimeValue.js

```diff
@@ -129,6 +248,97 @@ SELF.newFromJSON = function( json ) {
  */
 SELF.TYPE = 'time';
 
+// TODO: Inject configurations...
+/**
+ * Calendar configuration.
+ * @property {Object}
+ * @static
+ * @since 0.7
+ */
+SELF.CALENDARS = {
+	GREGORIAN: {
+		text: 'Gregorian',
```

Comment:
I would have called this "key" and made it lowercase. But I see that this is not new, it was like this before.